README.md

```diff
@@ -65,7 +65,8 @@ jobs:
     - uses: actions/checkout@v2
     - name: Setup tmate session
       uses: mxschmitt/action-tmate@v3
-      timeout-minutes: 15
+      with:
```

Comment:
Actually its also incorrect, its on the job level, but good catch!

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes

Reply:
Oops. I thought this was yet another input! My bad.

Reply:
And it is an option both on the job _and_ [on the step level](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes), so the original version is actually correct. I just need to lear how to read more carefully :smile: